src/GraphQL/Helper.php

```diff
@@ -157,6 +157,7 @@ public static function buildSqlCondition($defaultTable, $q, $op = null, $subject
                             $parts[] = '(' . self::quoteAbsoluteColumnName($defaultTable, $subject) . ' ' . $innerOp . ' ' . $db->quote($value) . ')';
                         }
                     } else {
+                        // TODO: Variable $fieldMappingTable and $key in isset() always exists and is not nullable.
                         if (isset($fieldMappingTable, $key)) {
```

Comment:
@fashxp Can you explain whats the purpose here? Should it be in_array or something?

Reply:
I think this must be right: https://github.com/pimcore/data-hub/pull/449/commits/75844da03124e65dc8077d3c06f53e17ee2e3cc6